docs/Auth-with-Ionic2.md

```diff
@@ -538,3 +543,31 @@ C:\projects\Ionic_AngularFire2_Project> ionic run android
 ```
 
 Once the App launches click on the Facebook login button and it should open up the native facebook app for authentication and once your enter credentials and gets succesfully authenticated, it should redirect you back to the home page.
+
+You should see your Facebook display name on Home page. 
+
+***Accessing Console Window while testing app on Mobile Phone.***
+
+Accessing the console window while running the app in browser is quite straighforward, but what if you want to access the console while testing your app on Mobile phone. Now that's pretty easy.
+
+Ensure that your mobile phone is connected to your computer and you've executed the following command (Basically your app is running on your mobile phone)
```

Comment:
Suggestion: "Ensure that your mobile phone is connected to your computer, you've executed the following command, and your app is now running on your device."